### PR TITLE
Content review round 1

### PR DESCRIPTION
**Pull request by miaodon**
Updated content as agreed in suggested screens.

### CODE REVIEW

File: conf/messages.en [hunk 1]
```diff
@@ -10,25 +10,22 @@ pageTitle.personal.accountDetails = Personal Account Details
 pageTitle.business.accountDetails = Business Account Details
 
 pageTitle.accountExistsIndeterminate = Check the account details
-content.accountExistsIndeterminate = We have not been able to check the account details $bankName$. Make sure the details you entered are correct.
+warning.accountExistsIndeterminate =
+content.accountExistsIndeterminate = We have not been able to check the account details with <bank name>. Make sure the details you entered are correct.
```
Comment (Ardesco) [inline conf/messages.en:14]: Should that be $bankName$ instead of <bank name>

File: conf/messages.en [hunk 2]
```diff
@@ -37,33 +34,33 @@ hint.accountNumber = Must be between 6 and 8 digits long
 hint.rollNumber = You can find it on your card, statement or passbook
 hint.companyRegistrationNumber = This should optionally begin with OC,LP,SC,SO,SL,NI,R,NC or NL followed by 6-8 digits
 
-error.summaryText = There is a problem
+error.summaryText = There is an problem
```
Comment (Ardesco) [inline conf/messages.en:37]: "There is **an** problem" doesn't sound right

Comment (miaodon) [reply conf/messages.en:37]: That's me, a mistype. Good spot. Please correct it.

File: conf/messages.en (hunk 1 above)
Comment (miaodon) [reply conf/messages.en:14]: It's just my placeholder, as I'd do to indicate dynamic content. Please feel free to change the way we indicate this.

**Comment by Ardesco** [comment]
Closed as covered by other PR's

**Pull request closed without merging**